src/jobs/models/jobsManager.ts

```diff
@@ -111,7 +120,25 @@ export class JobsManager {
         msg: `job & task completed - executing data merge from job metadata to record`,
       });
 
-      const mergedData = await this.mergeUpdatedRecord(job, task, isSwap);
+      this.logger.debug({
```

Comment:
insert this log into findRecord method

Reply:
i think it will break the simplified api wrap method. just for all extra details.

Reply:
we talk about it and understand that it will be added globally on future refactor task. to log all API calls from 3rd parties clients.

issue:
https://mapcolonies.atlassian.net/browse/MAPCO-4077